File: Capybara/Chapter1-Capybara+Android(web)OnSingleDevice/features/step_definitions/singleDevice.rb
  Given /^Open URL (.*)$/ do |url|
    visit url
  end

  When /^Enter username$/ do 
    page.find(:id,"userId").
    native.send_keys('test@pcloudy.com', :enter)
  end

  When /^Enter Password$/ do 
    page.find(:id,"password").
    native.send_keys('test1234', :enter)
  end

  When /^Click on login$/ do 
    page.find(:id,"loginSubmitBtn").
    click
  end

  

File: Selenide/Chapter3-Selenide+Android(Web)OnSingleDevice/src/test/java/org/selenide/examples/pcloudy/singleDevice.java
  package org.selenide.examples.pcloudy;

  import com.codeborne.selenide.Configuration;
  import com.codeborne.selenide.WebDriverRunner;
  import org.junit.jupiter.api.BeforeEach;
  import org.junit.jupiter.api.Test;
  import org.openqa.selenium.By;
  import org.openqa.selenium.support.events.AbstractWebDriverEventListener;

  import static com.codeborne.selenide.Condition.text;
  import static com.codeborne.selenide.Selenide.$;
  import static com.codeborne.selenide.Selenide.closeWebDriver;
  import static com.codeborne.selenide.Selenide.open;
  import java.io.*;

  
  public class singleDevice {

    @BeforeEach
    void setUp() {
      closeWebDriver();
      Configuration.browserSize = null;
      Configuration.browser = pcloudyconnection.class.getName();
      WebDriverRunner.addListener(new AbstractWebDriverEventListener() {
      });

      open();
    }

    @Test
    void Chrome() {
      open("https://device.pcloudy.com/login");

        $(By.id("userId"))
        .sendKeys("test@pcloudy.com");
        System.out.println("Username Entered");

        $(By.id("password"))
        .sendKeys("teat@123");;
        System.out.println("Password clicked");

        $(By.id("loginSubmitBtn"))
        .click();
        System.out.println("Click on login");
    }
  }
